notebooks/en/code_search.ipynb

```diff
@@ -0,0 +1,781 @@
+{
```

Comment:
<p>The doc-builder only supports <code>TIP</code> and <code>WARNING</code> so <code>IMPORTANT</code> won't work</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/151/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
Thanks, Steven. Fixed [`f9fbbb1` (#151)](https://github.com/huggingface/cookbook/pull/151/commits/f9fbbb1dd5aa64181390e0f7541419b8db14d9ae)